duckduckgo/iOS
# Updated content blocking implementation

Reviewer: Chris
Asana: https://app.asana.com/0/361428290920652/394268480131660

**Description**:
Remove content blocking categories from app, removes content blocking count from the omnibar (this will be replaced with a site rating soon) and adds a content blocking domain whitelist

**Steps to test this PR**:
1. Run the app and brows to a page
2. Disable content blocking and watch the logs to ensure that items are not blocked
3. Enable content blocking and watch the logs to ensure that items are blocked
4. Repeat the above this time enabling / disabling content blocking for a specific domain

###### Reviewer Checklist:


###### PR DRI Checklist:
